### DISCUSSION TITLE
Crusie Control disk capacity information?

### DISCUSSION THREAD

**Discussion by brads-au** [strimzi/strimzi-kafka-operator#9691] [Q&A]
Hi,

Just looking to get some more information on how Cruise Control detects the disk capacity configuration for the Kafka brokers. I've been seeing the following issue, however I can't quite work out how it is calculating the 256000.00 value. Also what unit is this value in, MB?

Strimzi version: 0.33.2 (so maybe just in need of an update)

[DiskCapacityGoal] Insufficient capacity for disk (Utilization
        5568257.00, Allowed Capacity 1228800.00, Threshold: 0.80). Add at least
        22 brokers with the same disk capacity (256000.00) as broker-0. Add at
        least 22 brokers with the same disk capacity (256000.00) as broker-0.'.

Cheers,
Brad

**Comment by scholzj** [comment]
The capacity is in MBs. I do not remember if there was any major change since 0.33, but the capacity we pass to Cruise Control is simply based on your storage configuration from the Kafka CR. How does your Kafka custom resource look like?
